platform.md

```diff
@@ -832,6 +832,12 @@ The following variables SHOULD be set in the lifecycle execution environment and
 
 The platform SHOULD NOT assume any other stack-provided environment variables are inherited by the buildpack.
 
+##### Builder-Provided Variables
```

Comment:
Is it possible for a platform to provide this variable even if it doesn't use builders?

Reply:
```suggestion
##### Platform-Provided Variables
```

I can't come up with something better, so I'm :+1: on that change.